src/temporal-logic/normalize_property.h

```diff
@@ -0,0 +1,16 @@
+/*******************************************************************\
+
+Module: Property Normalization
+
+Author: Daniel Kroening, dkr@amazon.com
+
+\*******************************************************************/
+
+#ifndef CPROVER_TEMPORAL_LOGIC_NORMALIZE_PROPERTY_H
+#define CPROVER_TEMPORAL_LOGIC_NORMALIZE_PROPERTY_H
+
+#include <util/expr.h>
+
+exprt normalize_property(exprt);
```

Comment:
Could you please add comments to describe what the normal form is that is being established?

Reply:
done